packages/twenty-server/src/workspace/workspace-query-builder/factories/delete-many-query.factory.ts

```diff
@@ -24,7 +24,7 @@ export class DeleteManyQueryFactory {
       mutation {
         deleteFrom${
           options.targetTableName
-        }Collection(filter: ${stringifyWithoutKeyQuote(args.filter)}) {
+        }Collection(filter: ${stringifyWithoutKeyQuote(args.filter)}, atMost: 30) {
```

Comment:
The addition of the 'atMost' parameter here is due to its default value being 1. If not set, the deletion of records beyond 1 will be blocked. However, I haven't decided how 'atMost' should be configured. Perhaps it should be passed in from the front end, similar to 'limit,' or it could be handled on the server side without the need for front-end awareness. I believe we need a brief discussion about this.

https://supabase.github.io/pg_graphql/api/#delete

Reply:
Interesting
@magrinj FYI